src/components/CommentForm.js

```diff
@@ -71,10 +73,15 @@ export default class CommentForm extends React.Component {
 
     const setText = event => updateStore(event.target.value)
     const placeholder = this.props.placeholder || 'Add a comment...'
-    const quickSubmit = onCmdEnter(e => {
-      e.preventDefault()
-      this.submit()
-    })
+    const keyDown = e => {
+      this.setState({enabled: this.refs.editor.getContent().length > 0})
+      onCmdEnter(e => {
+        e.preventDefault()
+        this.submit()
+      })(e)
```

Comment:
because onCmdEnter is curried, you can just pass the event as the second parameter instead of `()()`